tasks/discover-granules/package.json

```diff
@@ -38,9 +38,9 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@cumulus/common": "1.16.0",
-    "@cumulus/cumulus-message-adapter-js": "^1.1.0",
-    "@cumulus/ingest": "1.16.0"
+    "@cumulus/common": "1.16.1",
+    "@cumulus/cumulus-message-adapter-js": "^1.0.10",
```

Comment:
Ultimately both of these ranges will get any version below < 2.0.0, but not sure if you want to keep the previous range in place

Reply:
Yep you're right. Will fix everywhere.